tests/test_sourmash.py

```diff
@@ -3481,6 +3568,82 @@ def test_multigather_metagenome():
                     'NC_011294.1 Salmonella enterica subsp...' in out))
 
 
+@utils.in_tempdir
+def test_multigather_check_scaled_bounds_negative(c):
+    testdata_glob = utils.get_test_data('gather/GCF*.sig')
+    testdata_sigs = glob.glob(testdata_glob)
+
+    query_sig = utils.get_test_data('gather/combined.sig')
+
+    cmd = ['index', 'gcf_all']
+    cmd.extend(testdata_sigs)
+    cmd.extend(['-k', '21'])
+    c.run_sourmash(*cmd)
+
+    # make list w/query sig
```

Comment:
if you wanted to, you could switch from writing the `query.list` file and using `--query-from-file query.list`, to `--query query_sig` in the test. It would be slightly simpler.

Reply:
Okay, I'll try it out!